files/build_templates/sonic_debian_extension.j2

```diff
@@ -344,7 +344,14 @@ sudo chmod a+x $FILESYSTEM_ROOT/usr/sbin/policy-rc.d
 {% if installer_debs.strip() -%}
 {% for deb in installer_debs.strip().split(' ') -%}
 {% if sonic_asic_platform == "mellanox" %}
-sudo dpkg --extract {{deb}} $FILESYSTEM_ROOT
```

Comment:
What is the purpose? related to buster?

Reply:
check this pr. https://github.com/Azure/sonic-buildimage/pull/4405/commits/9b9e5c72640ff4c322e9a784ae63677e09584239

it is a workaround. somehow, the dpkg extract corrupt the root file system. I think mellanox own an official fix for this. they are working on this.